SDKTest/EventTests.swift

```diff
@@ -238,25 +243,28 @@ class EventTests: XCTestCase {
         }
     }
      
-    func testManuallyTargetRegister(){
+    func testManuallyTargetRegister() {
         NeuroID.start()
-        var testView = UITextView();
+        let testView = UITextView()
         testView.id = "wow"
         NeuroID.manuallyRegisterTarget(view: testView)
-        var events = DataStore.getAllEvents()
-        var validEvent = events.filter {  $0.type == "REGISTER_TARGET"}
+        let events = DataStore.getAllEvents()
+        let registeredTargetEvents = events.filter { $0.type == "REGISTER_TARGET" }
+        let validEvent = registeredTargetEvents.filter { $0.tgs == "wow" }
         assert(validEvent.count == 1)
+        assert(validEvent[0].et == "UITextView::UITextView")
```

Comment:
@cometman - This is the test that I found the `DataStore` build up on because the `validEvent` actually had 4 events and was failing. I checked and this was the case before my changes/refactoring. So to mitigate this I added the new `setup` block and then added an extra assert to make sure we registered the `UITextView` and not just some view